src/ui/mainwindow.cpp

```diff
@@ -471,6 +471,7 @@ MainWindow::~MainWindow()
     m_settings->windowGeometry = saveGeometry();
 
     delete ui;
+    delete ui->webView;
```

Comment:
Should this line before `delete ui` ? `ui` is already deleted at here. It should not be referenced after there. (This is implementation depenedency. Some implementation may clear it zero after `delete`)